ebpf-for-windows.sln

```diff
@@ -1,9 +1,8 @@
-
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.4.33205.214
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ebpfverifier", "external\ebpf-verifier\build\prevail.vcxproj", "{7D5B4E68-C0FA-3F86-9405-F6400219B440}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "prevail", "external\ebpf-verifier\build\prevail.vcxproj", "{7D5B4E68-C0FA-3F86-9405-F6400219B440}"
```

Comment:
This change seems unrelated to the rest of the PR?

Reply:
Reverted this. At some point we do need to updated the .sln as Visual Studio keeps on trying to modify it each time it is opened.